### PR TITLE
Rename terminal settings to terminal mode (side panel)

### PR DESCRIPTION
**Pull request by KievDevel**

### CODE REVIEW

**Review by greg-fer — APPROVED** [review]

File: src/features/history/effects.ts [hunk 1]
```diff
@@ -50,7 +50,7 @@ export const initializeHistoryBuffer: AppThunk<RootState> = async (
         dispatch(setNumberOfLinesInHistory(historyBuffer.getNumberOfLines()));
         if (dispatch(validateNumberOfLinesInHistory)) {
             logger.warn(
-                'History file is full, consider increasing the Max lines in history file, or cleaning it up, by using the Terminal Settings in the Side Panel.'
+                'History file is full, consider increasing the Max lines in history file, or cleaning it up, by using the Terminal Settings in the Side Panel.' // todo: is that correct message? How to clean it from side panel?
```
Comment (KievDevel) [inline src/features/history/effects.ts:53]: @aadnekar Please check this todo, I'm not sure message is correct

Comment (aadnekar) [reply src/features/history/effects.ts:53]: Yes, we removed that feature. So, the message is wrong.

Comment (aadnekar) [reply src/features/history/effects.ts:53]: image

Could just warn that the history file is full, and that more info can be found in the Terminal Settings tab?

```suggestion
                'History file is full. You can find more information about the history file in the Terminal Settings tab.'
```

Comment (KievDevel) [reply src/features/history/effects.ts]: Looks good, committed

**Review by aadnekar — APPROVED** [review]
Looks good now, but I think this one may have some conflicts with https://github.com/NordicSemiconductor/pc-nrfconnect-serial-terminal/pull/63

**Comment by greg-fer** [comment]
> Looks good now, but I think this one may have some conflicts with #63

Docs will be released after the app.

**Pull request merged**